server/sonar-web/src/main/js/api/webhooks.ts

```diff
@@ -60,3 +56,56 @@ export function updateWebhook(data: {
     .catch(() => Promise.resolve())
     .catch(throwGlobalError);
 }
+
+export function searchDeliveries(data: {
+  ceTaskId?: string;
+  componentKey?: string;
+  webhook?: string;
```

Comment:
why `webhook` is optional?

Reply:
Because we can ask for all deliveries related to a component for exemple, or a ce task. It matches the definition of the API.